helper/Data.php

```diff
@@ -187,6 +187,10 @@ public function encrypt($data)
 
     public function decrypt($data)
     {
+        if (empty($data)) {
+            return '';
```

Comment:
when should this be empty ? should we log this ?

Reply:
If something goes wrong I believe - last time it was empty because the cron job token was empty. Yes we can log this, could help with debugging.